api/lib/application/security-pre-handlers.js

```diff
@@ -145,6 +146,28 @@ async function checkUserIsAdminInOrganizationOrHasRolePixMaster(request, h) {
   return _replyWithAuthorizationError(h);
 }
 
+async function checkUserBelongsToOrganizationManagingStudents(request, h) {
+  if (!request.auth.credentials || !request.auth.credentials.userId) {
+    return _replyWithAuthorizationError(h);
+  }
+
+  const userId = request.auth.credentials.userId;
+  const organizationId = parseInt(request.params.id) || parseInt(request.payload.data.attributes['organization-id']);
+
+  let belongsToOrganizationManagingStudents;
+  try {
+    belongsToOrganizationManagingStudents = await checkUserBelongsToOrganizationManagingStudentsUseCase.execute(userId, organizationId);
+  } catch (err) {
+    return _replyWithAuthorizationError(h);
+  }
+
+  if (belongsToOrganizationManagingStudents) {
+    return h.response(true);
+  }
+
+  return _replyWithAuthorizationError(h);
```

Comment:
Je pense qu'on peut simplifier le code suivant : 
```suggestion
  try {
    if (await checkUserBelongsToOrganizationManagingStudentsUseCase.execute(userId, organizationId)) {
      return h.response(true);
    }
  } finally {
    return _replyWithAuthorizationError(h);
  }
```

Reply:
j'ai une erreur de linter en faisant ça : no-unsafe-finally, mais j'ai inliné quand même `belongsToOrganizationManagingStudents `